docs/tutorial/encryption.txt

```diff
@@ -19,16 +19,15 @@ To get started using in-use encryption in your project, the
 to be compiled with `libmongocrypt <https://github.com/mongodb/libmongocrypt>`_
 (enabled by default).
 
-Additionally, either `crypt_shared`_ or `mongocryptd`_ are required in order to
```

Comment:
I believe these might have been links at some point, which would explain the use of a trailing underscore. In any event, they're no longer defined.

I will point out that this docs page inconsistently uses code-formatting for crypt_shared and mongocryptd. I'm not sure what the server manual does, but we could probably stand to make this internally consistent. Feel free to make this consistent with whatever you do in docs beyond PHPLIB. I also have no objections to just removing all of the backticks for these terms.

Reply:
Looks like the Server manual formats them in monospace ([like on this page](https://www.mongodb.com/docs/v6.2/core/queryable-encryption/reference/mongocryptd/)), so I monospaced all instances of crypt_shared and mongocryptd